packages/js/test/plugins/auctionHouseModule/helpers.ts

```diff
@@ -1,15 +1,17 @@
 import type { Metaplex } from '@/Metaplex';
 import { CreateAuctionHouseInput } from '@/plugins';
-import { sol } from '@/types';
+import { sol, Signer } from '@/types';
 
 export const createAuctionHouse = async (
   mx: Metaplex,
+  auctioneerAuthority?: Signer,
```

Comment:
Since `auctioneerAuthority` is already in the `CreateAuctionHouseInput`, can we remove that second argument?

I think you did this because `mx.auctions().for(auctionHouse, auctioneerAuthority)` requires `auctioneerAuthority` to be a `Signer` but that was a mistake on my part. I thought at the time that you needed it to be a signer to make calls but actually, only the `authority` seem to matter as a Signer.

Would you mind downgrading `auctioneerAuthority` of the `for` method to a `PublicKey`? Everything should still work the same.

Reply:
Actually, the auctioneerAuthority is a Signer in Buy, Sell, and other auctioneer actions.
So, you were right in your assumptions.

Reply:
Ah okay, thanks for clarifying. Sorry about that, too much context switching haha.